tests/Resources/ti.app.test.js

```diff
@@ -320,6 +320,13 @@ describe('Titanium.App', function () {
 
 	it.android('pause/resume events', function (finish) {
 		this.timeout(5000);
+
+		// Cannot resume activity on Android 10.0+
```

Comment:
Seems like something we should add in our docs at the very least. Are we sure this is a hard OS limitation, or is there a fix we need?

Reply:
Yes, this is a well known limitation Google introduced as of Android 10. The idea is you don't want some rogue backgrounded app to display a window without user interaction. Pretty much the equivalent of browser popups. iOS has never supported this and I'm not aware of customer apps that do this. (It's considered extremely naughty.)
https://developer.android.com/guide/components/activities/background-starts

Perhaps a test like this is best done by QE's UI integration suite, where I would hope we can run the same test for iOS.

Our "ti.android.test.js" script's "activity callback" unit test will fail on Android 10 and above for the same reason. Although, we should be able to keep that unit test by having it open/close a child window instead of navigating to the home screen.
https://github.com/appcelerator/titanium_mobile/blob/b21c5d79d00e6efad06fcea441712f57c3bbaa01/tests/Resources/ti.android.test.js#L65